src/query/serial.c

```diff
@@ -1486,8 +1491,8 @@ serial_cache_index_btid (THREAD_ENTRY * thread_p)
   oid_get_serial_oid (&serial_oid);
   assert (!OID_ISNULL (&serial_oid));
 
-  /* Now try to get index BTID. Serial index name is "pk_db_serial_name". */
-  error_code = heap_get_btid_from_index_name (thread_p, &serial_oid, "pk_db_serial_name", &serial_Cached_btid);
+  /* Now try to get index BTID. Serial index name is "pk_db_serial_name_owner". */
+  error_code = heap_get_btid_from_index_name (thread_p, &serial_oid, "u_db_serial_full_name", &serial_Cached_btid);
```

Comment:
According to the comment, shouldn't we use "pk_db_serial_name_owner"?

Reply:
I'm missing a comment change. I'll remove it from the comments.